drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/test/java/org/drools/workbench/screens/scenariosimulation/backend/server/ScenarioSimulationServiceImplTest.java

```diff
@@ -404,4 +421,20 @@ public void editPomIfNecessaryTest() {
     public void getActivatorPathTest() {
         assertTrue(service.getActivatorPath(packageMock).endsWith(ScenarioJunitActivator.ACTIVATOR_CLASS_NAME + ".java"));
     }
+
+    @Test
+    public void load() {
+        ScenarioSimulationModel model = service.load(path);
+
+        assertEquals(Type.DMN, model.getSimulation().getSimulationDescriptor().getType());
+        verify(dmnTypeServiceMock, times(1)).initializeNameAndNamespace(any(), any(), anyString());
+
+        when(dmnTypeServiceMock.initializeNameAndNamespace(any(), any(), anyString())).thenThrow(new ImpossibleToFindDMNException(""));
+
+        try {
+            service.load(path);
+        } catch (Exception e) {
+            fail();
+        }
```

Comment:
Maybe I miss something, why this test invoked the method `load` twice? Shouldn't we rather assert the loaded `model` content?

Reply:
@danielezonca any comments to this please?

Reply:
Sorry I missed you comment :)

I didn't create a new method but it covers a different code path. It just verifies that the exception is not propagated

Reply:
Thank you